### PR TITLE
Run3-gex124E Try to remove compilation warnings in RecoParticleFlow/PFTracking

### PR DESCRIPTION
**Pull request by bsunanda**
#### PR description:

Try to remove compilation warnings in RecoParticleFlow/PFTracking

#### PR validation:

Use the runTheMatrix test workflows

#### if this PR is a backport please specify the original PR and why you need to backport that PR:

Nothing special

### CODE REVIEW

**Comment by bsunanda** [comment]
@cmsbuild Please test

File: RecoParticleFlow/PFTracking/interface/PFCheckHitPattern.h [hunk 1]
```diff
@@ -4,7 +4,6 @@
 // standard EDAnalyser include files
```
Comment (clacaputo) [inline RecoParticleFlow/PFTracking/interface/PFCheckHitPattern.h:4]: Could you please modify https://github.com/cms-sw/cmssw/blob/feb29957faf4be06f28e72d2a76eef03dea1beaf/RecoParticleFlow/PFTracking/interface/PFCheckHitPattern.h#L1-L2

to

```cpp
#ifndef RecoParticleFlow_PFTracking_PFCheckHitPattern_H
#define RecoParticleFlow_PFTracking_PFCheckHitPattern_H
```

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Comment by clacaputo** [comment]
+reconstruction

- technical PR
- no reco changes expected or observed

**Comment by qliphy** [comment]
+1

**Pull request merged**